src/Appwrite/Event/Mail.php

```diff
@@ -118,7 +118,7 @@ public function getBody(): string
      */
     public function setName(string $name): self
     {
-        $this->name = $name;
+        $this->name = $name ?? '';
```

Comment:
I don't think this will work because according to the typing on the function, `$name` is a string and will never be `null`